### ISSUE TITLE
Secure old protocols one more time

### ISSUE THREAD

**Issue by adrianmroz** [allegro/turnilo#410]
@a2l007, let's create new issue for #128 

We merged that patch long time ago but we changed that behaviour in #346 and #362 

I will investigate if we didn't make any regression, but if you have counter example on hand, please post it.

**Comment by adrianmroz** [comment]
Creating druid requester should work as before, with new `url` field as with legacy `host`. For sure we could fix some documentation in `config.ts`

**Comment by a2l007** [comment]
Hello @adrianmroz , Thanks for checking this. I wasn't aware of the change in behaviour introduced in the patches that you mentioned. After using `url` instead of `host`, it is working successfully. We should be good to close this issue.

**Comment by adrianmroz** [comment]
We will close after #413 is merged. Documentation is a little muddy now! 

`url` is a way to go, we deprecate `host`/`protocol` pair.
